modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -0,0 +1,138 @@
+
+=== Mule Flow Generated by APIKit and used for the routing of APIKit Requests
+
+APIkit is a tool that simplifies the implementation of APIs by automatically generating a minimal set of Mule flows based on the API specification. Each APIKit router endpoint counts as a distinct Mule flow. These Mule flows do not have an event source and are used for the handling of HTTP requests for a particular method and path of an API.
+
+==== Example of APIKit Request:
+
+Example 1.b - This flow is used for the routing of APIKit requests. In this case, the flow handles the GET request in the `/reservation` path.
+
+[source,xml]
+----
+<flow name="get:\reservation:cocheras-puerto-madero-api-config">
+        <logger level="INFO" message="#[output json --- attributes.queryParams]" />
+</flow>
+----
+
+// Example Use-case [12 flows]: New customer profile data needs to be moved from Marketing Cloud to Master Data Management (MDM) 
+
+// Mule Flow with Event Source [1]: Main Mule Flow with the HTTP listener that routes message to appropriate flow generated by APIKit based on the message content.
+
+// image 
+
+// Mule Flows used for routing APIKit Requests [11]:
+
+// image
+
+[[non-billable-flows]]
+=== Mule Flows that do not Count against your Anypoint Platform Package Allotment 
+
+Mule Flows that do not have an event source and are not used for the routing of APIKit requests are not charged against your Anypoint Platform package allotment. These are Mule flows primarily used for modularization of code, or code reuse purposes.
+
+Example:
+
+[source,xml]
+----
+2.a - Flow with only a logger component
+<flow name="just-logging">
+        <logger level="INFO" message="#[output json --- attributes.queryParams]" />
+</flow>
+----
+
+[[mule-messages]]
+== Mule Messages
+
+A _Mule message_ is the data (the payload and its attributes) that passes through an application through one or more Mule flows. A Mule message is a part of a Mule Event, which is generated when the event source within a Mule flow of a Mule application is triggered. For instance, when an HTTP listener receives a request or each time the scheduler component triggers an execution of the Mule flow, a Mule event, consisting of a Mule message is created. 
+Mule message processors in a Mule flow (such as core components, file read operations, or the HTTP request operations) can then retrieve, set, and process Mule message data that resides in the Mule event according to their configurations. 
+A Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a Mule message returns a new Mule message consisting of a message payload (the body of the message) and message attributes (metadata associated with the message).
```

Comment:
```suggestion
A Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a Mule message returns a new Mule message that consists of a message payload (the body of the message) and message attributes (metadata associated with the message).
```

Reply:
I would break up this large paragraph by having this text as a separate paragraph.

Reply:
immutable > unchangeable?